src/_testing_/calculate.test.js

```diff
@@ -0,0 +1,151 @@
+/*eslint-disable*/
```

Comment:
- [ ] It will be great to fix the linter errors instead of disabling the linters. Remember linters are there to help you write clean code